<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGadgetWindow</class>
 <widget class="QDialog" name="AddGadgetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Gadget</string>
  </property>
  <property name="styleSheet">
   <string>QDialog {
	background-color: rgb(201, 226, 252);
}
QWidget#contentWidget {
	background-color: white;
}
QFrame#buttonBoxFrame {
	border-top: 1px solid #B8C6D9;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Choose a Gadget from the list below:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cmbGadgets"/>
        </item>
        <item>
         <widget class="QToolButton" name="cmdReload">
          <property name="icon">
           <iconset resource="../main.qrc">
            <normaloff>:/gui/icons/view-refresh.png</normaloff>:/gui/icons/view-refresh.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDescription">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonBoxFrame">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddGadgetWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddGadgetWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
